auth_signup_verify_email/controllers/main.py

```diff
@@ -34,7 +34,7 @@ def passwordless_signup(self, values):
             values["email"] = values.get("login")
 
         # Remove password
-        values["password"] = False
+        values["password"] = ""
```

Comment:
If we use the module as is with the password set to `False`, we can't signup:

```
Traceback (most recent call last):
  File "/Users/alex/Documents/workspace/odoo/addons/10.0/server-tools/auth_signup_verify_email/controllers/main.py", line 43, in passwordless_signup
    sudo_users.signup(values, qcontext.get("token"))
  File "/Users/alex/Documents/workspace/odoo/odoo10/addons/auth_signup/models/res_users.py", line 71, in signup
    self._signup_create_user(values)
  File "/Users/alex/Documents/workspace/odoo/odoo10/addons/auth_signup/models/res_users.py", line 98, in _signup_create_user
    raise SignupError(ustr(e))
SignupError: secret must be unicode or bytes, not bool
```

Can't find why this is raised, but, as stated in the code for the password field: "Keep empty if you don't want the user to be able to connect on the system." and the default value is `''` not `False`

I think even for odoo 9, the password should be set to `''`